src/main/java/com/limechain/network/PeerMessageCoordinator.java

```diff
@@ -55,6 +61,20 @@ public void sendBlockAnnounceMessageExcludingPeer(BlockAnnounceMessage message,
         );
     }
 
+    public void sendTransactionMessageExcludingPeer(Extrinsic extrinsic, Set<PeerId> excludingSet) {
+        ExtrinsicArray extrinsicArray = new ExtrinsicArray(new Extrinsic[]{extrinsic});
+        byte[] scaleMessage = ScaleUtils.Encode.encode(new TransactionWriter(), extrinsicArray);
+
+        sendMessageToActivePeers(p -> {
+            if (excludingSet.contains(p)) {
+                return;
+            }
+            asyncExecutor.executeAndForget(() -> network.getTransactionsService().sendTransactionsMessage(
+                    network.getHost(), p, scaleMessage
+            ));
+        });
+    }
```

Comment:
Can you add a short javadoc type of description to the function like in the `ConnectionManager` class

Reply:
Added